src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IFixedStatement.cs

```diff
@@ -365,5 +365,294 @@ void M1()
             VerifyOperationTreeAndDiagnosticsForTest<FixedStatementSyntax>(source, expectedOperationTree, expectedDiagnostics,
                 compilationOptions: TestOptions.UnsafeDebugDll);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void FixedStatement_01()
+        {
+            string source = @"
+unsafe public class MyClass
+{
+    int i;
+    unsafe void M(bool b)
+    /*<bind>*/{
+        fixed (int* p = &i)
+        {
+            System.Console.WriteLine($""P is {p}"");
```

Comment:
>System.Console.WriteLine($""P is {p}"") [](start = 12, length = 39)

Not required to make changes to the existing tests, but I would recommend to use very simple expressions in order to keep base-lines small. A simple assignment would work, for example.

Reply:
Noted.

---
In reply to: [168006425](https://github.com/dotnet/roslyn/pull/24792#discussion_r168006425) [](ancestors = 168006425)